database/index.js

```diff
@@ -7,6 +7,7 @@ mongoose.connect(
 );
 
 let bookSchema = mongoose.Schema({
+  relatedIsbn: Number,
   isbn: Number,
```

Comment:
I checked and it seems that ISBN should be a string and not a number because it might start with zero.

Reply:
Changed ISBN from number to string, and re-seeded my DB w/ this change.